app/presenters/result_set_presenter.rb

```diff
@@ -28,23 +28,15 @@ def search_results_content
       zero_results: total.zero?,
       page_count: documents.count,
       finder_name: finder.name,
-      debug_score: debug_score,
+      debug_score: debug_score
     }
   end
 
   def documents
     @documents ||= begin
```

Comment:
this isn't really a document anymore. Could we rename it to something more suitable?

Reply:
What is it a list of?

Reply:
We'll look at this in a separate refactor.